docs/quantize/activation_quantization.rst

```diff
@@ -0,0 +1,47 @@
+Activation quantization
+=======================
+
+Blueoil quantize an activation function when the last convolution layer is quantized. (``quantize_last_conv`` =True), otherwise ``Leaky ReLU`` is used as an activation function.
```

Comment:
Let me check my understand, If `quantize_last_conv = True` What happen?
`quantize_last_convolution` variable is really used?

Reply:
Sorry, my typo the variable name should be `quantize_last_convolution`

I think it used in quantized network class constructor.
 ```
def __init__( ..., quantize_last_convolution, ):
        self.quantize_last_convolution = quantize_last_convolution
        ...
        self.activation = activation_quantizer(**activation_quantizer_kwargs)
        ...
        if self.quantize_last_convolution:
                self.before_last_activation = self.activation
        else:
                self.before_last_activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
 ```

Ref:
`lm_fyolo.py` https://github.com/blue-oil/blueoil/blob/ac416513e1ba8fccd92742b68e9c8a807f1e54ff/blueoil/networks/object_detection/lm_fyolo.py#L188
`darknet.py` https://github.com/blue-oil/blueoil/blob/ac416513e1ba8fccd92742b68e9c8a807f1e54ff/blueoil/networks/classification/darknet.py#L306

Reply:
Oh I found that, the current config file did not contain `quantize_last_convolution`, I will check this again.